components/footer/index.js

```diff
@@ -1,7 +1,7 @@
 export const Footer = () => {
   return (
     <footer>
-      The contents of this website are deployed from this{' '}
+      The contents of this website are deployed on this&nbsp;
```

Comment:
Don't you think the previous statement made more sense?

Reply:
Now that I re-read it, it does make sense over what I wrote. Let me update the PR :)

Reply:
Now that I re-read it, it does make more sense. Let me update the PR :)